core/domain/change_domain.py

```diff
@@ -222,3 +222,6 @@ def to_dict(self):
                     self, attribute_name)
 
         return base_change_dict
+
+    def __getattr__(self, name: str) -> str:
+        return self.__dict__[name]
```

Comment:
Sorry, actually you will have to do this
```suggestion
        # AttributeError needs to be thrown in order to make 
        # instances of this class picklable.
        try:
            return self.__dict__[name]
        except KeyError:
	    raise AttributeError(name)
```

Reply:
@vojtechjelinek done

Reply:
Not done, you didn't add the explanatory comment.

Reply:
Oh, right I am sorry. Added the comments now @vojtechjelinek. Please take a look.